central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -84,11 +74,32 @@ func newSentinelNotifier(notifier *storage.Notifier) (*sentinel, error) {
 	}
 
 	return &sentinel{
-		notifier:     notifier,
-		azlogsClient: &azureLogsClientImpl{client: client},
+		notifier:             notifier,
+		azlogsClient:         &azureLogsClientImpl{client: client},
+		azureMaxLogFieldSize: azureFieldSizeLimit,
 	}, nil
 }
 
+func (s sentinel) SendAuditMessage(ctx context.Context, msg *v1.Audit_Message) error {
+	if !features.MicrosoftSentinelNotifier.Enabled() {
```

Comment:
combine both if conditions into an or?

Reply:
I've not done it intentionally because when the feature flag is removed, the if can just be removed.